src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random rnd = new Random();
```

Comment:
why static?
```suggestion
    private static final Random random = new Random();
```

rnd -> not informative variable name, let's not use abbreviations